test/functional/apps/dashboard/dashboard_filtering.js

```diff
@@ -41,6 +42,7 @@ export default function({ getService, getPageObjects }) {
 
     before(async () => {
       await esArchiver.load('dashboard/current/kibana');
+      await security.testUser.setRoles(['kibana_admin', 'test_logstash_reader', 'animals']);
```

Comment:
Missing the `restoreDefaults()` call here.

Reply:
added